packages/monitoring-system/data-processor/test/unit/data-processors/cloud-logs-data-processor.spec.ts

```diff
@@ -563,15 +757,23 @@ describe('Cloud Logs Processor', () => {
         'execution-start.json',
       ]);
 
-      it('calls nack() on message if there is an error in processing and logs it', () => {
+      it('calls nack() on message for processing error, and rethrows error', () => {
```

Comment:
Changed specs so that runtime errors are `nack`ed and rethrown rather than quietly logged